engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -271,6 +271,21 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
       }
     }
 
+  def versionRoute =
+    path("engine" / Segment / "version") { version =>
+      get {
+        complete {
+          val versionConf = ConfigFactory.load("cromwell-version.conf").getConfig("version")
```

Comment:
maybe `lazy val` this as a class member so it doesn't have to be loaded every time somebody hits the endpoint